src/documents/showcase/Blasting Agent꞉ Ultimate Edition.md

```diff
@@ -4,6 +4,7 @@ title: "Blasting Agent꞉ Ultimate Edition"
 windows: http://www.blastingagent.com/
 steam: http://store.steampowered.com/app/453810
 ps4: https://store.playstation.com/en-us/product/UP0891-CUSA06555_00-RGBLASTINGAGENTU
-website: http://www.blastingagent.com/
-date: "5-10-2016"
+itch: https://axolstudio.itch.io/blasting-agent
+website: https://axolstudio.com/projects/blasting-agent/
+date: "8-17-2016"
```

Comment:
whats this date change based on?

Reply:
Neither of those dates are accurate, really...

That August date is the Steam release date.

It was released originally on Jan 07, 2016 on Itch.io (I released it on my Birthday so I would remember)

Then it was released on consoles later on.

Reply:
@ninjamuffin99 you got some 'splainin to do